### DISCUSSION TITLE
Add MySQL support

### DISCUSSION THREAD

**Discussion by 0xPoe** [tokio-rs/toasty#37] [Ideas]
I’m very interested in expanding it to include additional SQL databases.

@carllerche Do you have any suggestions? Should I try it now, or wait until the driver API and other APIs are more stable? Additionally, do you have any guidance or recommended steps for adding a new database?

I also have a question about dependencies. It seems we’re using `rusqlite` as the dependency for the SQLite driver. If we want to support more databases, should we consider switching to `sqlx` to cover most databases?

**Comment by carllerche** [comment]
Someone in Discord is working on implementing `toasty-sqlx` to unblock more database support: https://discord.com/channels/500028886025895936/1300487772301168640
